api/routes/web.py

```diff
@@ -34,7 +34,9 @@ class ProjectSummaryResponse(BaseModel):
     # high level stats
     total_participants: int
     total_samples: int
-    sequence_stats: dict[str, dict[str, str]]
+    total_sequences: int
+    cram_seqr_stats: dict[str, dict[str, str]]
```

Comment:
Could be worth a comment here just saying _by sequence_type then batch_

Reply:
Added comments - cram_seqr_stats is by seqType then cram/seqr/seq counts, batch_sequence_stats is by batch then seqType.